pkgs/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_function_declaration.dart

```diff
@@ -91,6 +91,22 @@ ParsedFunctionInfo parseFunctionInfo(
 
   final openParen = tokens.indexWhere((tok) => matchFragment(tok, 'text', '('));
   if (openParen == -1) throw malformedInitializerException;
+
+  final prefixAnnotations = <String>{};
+  while (true) {
+    final tok = tokens.first['kind'].get<String>();
+    if (tok == 'keyword') {
+      final keyword = maybeConsume('keyword');
+      if (keyword == 'func') {
+        break;
+      } else if (keyword == null) {
```

Comment:
I don't think this case will ever be hit. `maybeConsume` only returns null if the `kind` doesn't match, but this whole section is inside an if statement that's already checking that the kind is `'keyword'`. So you can delete this case.